### PR TITLE
Use spring-boot-template with small customisations

### PR DESCRIPTION
**Pull request by KarolJastrzebski**
- Renamed package: demo -> bulkscanning
- Removed docker-related files as the architecture is not known yet.
- Wrote minimalistic README.md

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscanning/controllers/RootController.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+     *
+     * <p>Azure application service has a hidden feature of making requests to root endpoint when
+     * "Always On" is turned on.
+     * This is the endpoint to deal with that and therefore silence the unnecessary 404s as a response code.
+     *
+     * @return Welcome message from the service.
+     */
+    @GetMapping
+    public ResponseEntity<String> welcome() {
+        return ok("Welcome to spring-boot-template");
```
Comment (doncem) [inline src/main/java/uk/gov/hmcts/reform/bulkscanning/controllers/RootController.java]: oopsie

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+server:
+  port: 4550
```
Comment (doncem) [inline src/main/resources/application.yaml]: here too

File: .env [hunk 3]
```diff
@@ -0,0 +1 @@
+SERVER_PORT=4550
```
Comment (doncem) [inline .env]: what will the port number be?

File: src/test/java/uk/gov/hmcts/reform/bulkscanning/controllers/GetWelcomeTest.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+public class GetWelcomeTest {
+
+    @Autowired
+    private transient MockMvc mockMvc;
+
+    @Test
+    public void should_welcome_upon_root_request_with_200_response_code() throws Exception {
+        MvcResult response = mockMvc.perform(get("/")).andExpect(status().isOk()).andReturn();
+
+        assertThat(response.getResponse().getContentAsString()).startsWith("Welcome");
```
Comment (nitinprabhu) [inline src/test/java/uk/gov/hmcts/reform/bulkscanning/controllers/GetWelcomeTest.java]: You could Assert with complete String :)

File: web.config [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (nitinprabhu) [inline web.config]: Platform config can be done as a separate PR.This way it would be very clear to understand.

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanning/ApplicationTests.java [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class ApplicationTests {
+
+    @Test
+    public void sample_test() {
+        assertThat(2).isGreaterThan(1);
```
Comment (nitinprabhu) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscanning/ApplicationTests.java]: Not sure why we want this ?

File: .travis.yml [hunk 7]
```diff
@@ -0,0 +1,29 @@
+language: java
```
Comment (nitinprabhu) [inline .travis.yml]: Enabling Travis could be a separate PR

File: .env (hunk 3 above)
Comment (nitinprabhu) [inline .env]: This file could be added as part of Docker PR

File: .travis.yml (hunk 7 above)
Comment (KarolJastrzebski) [reply .travis.yml]: True, but it's simple enough to have it as a base, even if it's only for running tests and jacoco.
